SWARAM-UI-Documentation/components.css

```
@@ -0,0 +1,512 @@
+    --font-size-text:1.1rem;
+    --font-size-small:0.8rem;
+    --font-size-title:01.4rem;
+    --font-family:Roboto;
+    --margin-right:0.6rem;
+}
+
+body{
+    margin:0;
+}
+
+.bg-primary{
+    background-color: var(--primary);
+    color:white;
+}
+
+.text-color-primary{
+    color:var(--primary)
+}
+
+.flex-horizontal{
+    display: flex;
+}
+
+/** Avatar **/
+
+.avatar-container{
+    position: relative;
+}
+
+.img-size-medium{
+    width:5.5rem;
+    height: 5.5rem;
+}
+
+.img-size-large{
+    width:7.5rem;
+    height: 7.5rem;
+}
+
+.img-size-small{
+    width:3.5rem;
+    height: 3.5rem;
+}
+.img-size-xs{
+    width:2rem;
+    height: 2rem;
+}
+
+/** Alerts **/
+
+.alert{
+    padding:var(--padding-2) 0rem;
+    padding-left:1rem;
+    margin-bottom: 1rem;
+    font-size:var(--font-size-text);
+    position: relative;  
+    border-radius:0.3rem
+}
+
+.alert-success{
+    background-color:#A7F3D0;
+    color:#0f5132;
+}
+
+.alert-error{
+    background-color:#FECACA;
+    color:#842029;
+}
+
+.alert-warning{
+    background-color: #f3e096;
+    color:#664d03;
+}
+
+/** Badges **/
+
+.badge{  
+    position:absolute;
+    text-align:center;
+}
+
+.badge-circle{
+    border-radius:50%; 
+    padding:0.1rem 0.4rem; 
+    font-size:var(--font-size-small);
+    font-weight: 700; 
+}
+
+.badge-rect{
+    padding:0rem 0.3rem;
+}
+
+.badge-anchorTopRight{  
+    top:0;
+    right:0;
+}
+
+.badge-anchorTopLeft{  
+    top:0.3rem;
+    left:0;
+}
+
+.badge-anchorBottomRight{
+    bottom:0;
+    right:0;
+}
+
+.badge-dot{
+    position:absolute;
+    background-color: #059669;
+    border:2px solid white;
+    border-radius:50%;
+    min-width:0.7rem;
+    min-height:0.7rem;   
+}
+
+/** Buttons **/
+
+.btn{
+    line-height:1.3rem;
+    border-radius:0.25rem;   
+    padding:var(--padding-1) var(--padding-2);
+    font-weight: 550;
+    font-size:0.9rem;
+    cursor: pointer;
+    outline:none;
+    text-align: center;
+} 
+
+.btn-primary-contained{
+    background-color:var(--primary);
+    border:1px solid var(--primary);
+    color:white;
+}
+
+.btn-primary-contained:hover{
+    background-color: #991B1B;
+}
+
+.btn-primary-outline{
+    color:var(--primary);
+    background-color: white;
+    border:1px solid var(--primary);
+}
+
+.btn-text{
+    border:0;
+    background-color: white;
+    padding:0; 
+}
+/** Icon Buttons **/
+
+.icon-btn{
+    border: 0;
+    outline:none; 
+    position: relative; 
+    background-color:transparent;  
+    padding:0 0.2rem;
+}
+
+.icon-btn-with-padding{
+    border: 0;
+    outline:none; 
+    padding:0.7rem 0.4rem;
+    position: relative; 
+    background-color:transparent;  
+}
+
+.icon-size-36{
+    font-size: 36px; 
+}
+
+.icon-size-18{
+    font-size: 18px; 
+}
+
+.icon-color-red{
+    color:red;
+}
+
+.icon-color-primary{
+    color:var(--primary)
+}
+.icon-color-white{
+    color:white;
+}
+
+.icon-color-gray{
+    color:gray;
+}
+
+/**Floating action button**/
+
+.floating-action-btn{
```

The floating button + icon seems a bit off in top-right direction, try giving it display: flex and then adjusting it in center vertically and horizontally, that should fix it.

also you can directly give top and right 2rem to avoid giving margin explicitly to the button.

```suggestion
.floating-action-btn{
  width: 3rem;
  height: 3rem;
  border: none;
  outline: none;
  border-radius: 50%;
  position: fixed;
  z-index: 1;
  bottom: 2rem;
  right: 2rem;
  /* margin: 2rem; */
  display: flex;
  align-items: center;
  justify-content: center;}
```

---

Thank you for the suggestion.Will fix it

---

I gave margin to the floating action button alone because it is fixed and does not depend on any other element